network/stream/stream.go

```diff
@@ -679,6 +679,11 @@ func (r *Registry) createPriceOracle() {
 			PerByte: true,
 			Payer:   protocols.Receiver,
 		},
+		reflect.TypeOf(ChunkDeliveryMsgSyncing{}): {
```

Comment:
Why does this have a price? This would mean syncing is also accounted for in swap, but it should not. It will be accounted for with postage stamps

Reply:
It probably shouldn't. It's only here for testing (this PR is not meant to be merged). The problem is in a small swarm network (2 nodes) all files basically sync instantly and then every request is simply from local storage and it's a bit hard to trigger a ChunkDeliveryMsgRetrieval.

Reply:
oh I see